frontend/cypress/integration/common/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+
+And('user selects the {string} namespace in the NamespaceSelector', (namespace: string) => {
+  cy.getBySel('namespace dropdown').click();
+  cy.get('input[type="checkbox"][value="bookinfo"]').click();
+  cy.getBySel('namespace dropdown').click();
+});
```

Comment:
Not totally sure where to put this and the `navigation` step definitions. My thought was to group generic step definitions by function. In this case anything generic related to namespaces could go here. Same for `navigation.ts`.